### ISSUE TITLE
Update to TensorFlow 1.13

### ISSUE THREAD

**Issue by deliahu** [cortexlabs/cortex#95]
### Motivation

There is a [bug](https://github.com/tensorflow/tensorflow/issues/25175) in TensorFlow 1.12 which breaks dropout layers in Keras.
